localstack/deprecations.py

```diff
@@ -47,12 +47,6 @@ def is_affected(self) -> bool:
         "0.12.8",
         "PORT_WEB_UI has been removed, and is not available anymore. Please remove this environment variable.",
     ),
-    # Since 0.12.7 - Full multi-region support
-    EnvVarDeprecation(
-        "USE_SINGLE_REGION",
-        "0.12.7",
-        "LocalStack now has full multi-region support. Please remove this environment variable.",
-    ),
```

Comment:
I think we should keep this deprecation warning, but just slightly change the text to something like "LocalStack now has full multi-region support.This variable has been removed and does not have any effect anymore. Please remove it form your configuration."